components/Code.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Icon } from '@iconify/react';
```

Comment:
don't use a package in this fix

use an SVG component just like `<CopyIcon />` is implemented

You can get the Check Icon [here](https://feathericons.com/?query=check)